addons/mail/static/src/components/autocomplete_input_view/autocomplete_input_view.js

```diff
@@ -0,0 +1,78 @@
+/** @odoo-module **/
+
+import { useComponentToModel } from '@mail/component_hooks/use_component_to_model';
+import { registerMessagingComponent } from '@mail/utils/messaging_component';
+
+const { Component, onMounted, onWillUnmount } = owl;
+
+export class AutocompleteInputView extends Component {
+
+    /**
+     * @override
+     */
+    setup() {
+        super.setup();
+        useComponentToModel({ fieldName: 'component' });
+        onMounted(() => this._mounted());
+        onWillUnmount(() => this._willUnmount());
+    }
+
+    _mounted() {
+        if (!this.root.el) {
+            return;
+        }
+        if (this.autocompleteInputView.isFocusOnMount) {
+            this.root.el.focus();
+        }
+
+        let args = {
+            autoFocus: true,
+            select: (ev, ui) => {
+                if (this.autocompleteInputView) {
+                    this.autocompleteInputView.onSelect(ev, ui);
+                }
+            },
+            source: (req, res) => {
+                if (this.autocompleteInputView) {
+                    this.autocompleteInputView.onSource(req, res);
+                }
+            },
+            focus: ev => ev.preventDefault(),
```

Comment:
do you know why it is needed to preventDefault the focus event? it would be great either to add some comment explaining why this is important, or on the contrary to delete it if it turns out to be useless